src/registration/InterestInTerraCheckbox.tsx

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash/fp';
+import React from 'react';
+import { LabeledCheckbox } from 'src/components/common';
+
+interface InterestInTerraCheckboxProps {
+  title: string;
+  interestInTerra: string;
+  setFunc: (interest: string) => void;
+}
+
+export const InterestInTerraCheckbox = (props: InterestInTerraCheckboxProps) => (
+  <div style={{ marginTop: '.25rem' }}>
+    <LabeledCheckbox
+      checked={_.includes(props.title, props.interestInTerra)}
+      disabled={false}
+      onChange={(v: string) => {
```

Comment:
LabeledCheckbox's onChange function takes a boolean (the current checked state of the checkbox).

```suggestion
      onChange={(v: boolean) => {
```